test/src/main/java/org/apache/accumulo/test/ShellServerIT.java

```diff
@@ -1737,60 +1737,112 @@ public void scansWithClassLoaderContext() throws Exception {
       assertTrue(true);
     }
     ts.exec("createtable t");
+    // Assert that the TabletServer does not know anything about our class
+    String result = ts.exec("setiter -scan -n reverse -t t -p 21 -class org.apache.accumulo.test.functional.ValueReversingIterator");
```

Comment:
I realize this is existing code you did not write, but if you have a second while you are here it would be nice to make this class name a constant.